applications.json

```diff
@@ -22,6 +22,17 @@
       "github_URL": "https://github.com/marsprotocol",
       "featured": false
     },
+    {
+      "title": "Mars Farm",
+      "subtitle": "Farm the fields of Mars. Leveraged yield farming on top of Apollo auto compounding vaults! Proceed with caution fellow farmer. Riches and ruins lie ahead.",
+      "external_URL": "https://mars.osmosis.zone/farm",
+      "thumbnail_image_URL": "????",
+      "hero_image_URL": "????",
+      "twitter_URL": "https://twitter.com/mars_protocol",
+      "medium_URL": "https://mars-protocol.medium.com/",
+      "github_URL": "https://github.com/mars-protocol",
```

Comment:
Would this be the farm-specific repo?
https://github.com/mars-protocol/v2-fields-of-mars